extract-url.js

```
@@ -27,9 +27,8 @@ const searchForKeywords = async (url ) => {
       try {
       await driver.get(`${url}?search=${keyword.split(" ").join('+')}`);
```

is there any effect to this line? it looks like it discards the result and if an error happens it gets silently ignored